### PR TITLE
reverted changes to the assets, referenced to the input tag

### PR DESCRIPTION
**Pull request by yash-zededa**
Reverted changes to the assets.yml added the reference and workflow call from the publish.yml

### CODE REVIEW

**Review by eriknordmark — APPROVED** [review]
LGTM

File: .github/workflows/assets.yml [hunk 1]
```diff
@@ -47,56 +38,57 @@ jobs:
           # if the default server is responding -- we can skip apt update
           $APT_INSTALL || { sudo apt update && $APT_INSTALL ; }
           echo "ARCH=${{ matrix.arch }}" >> "$GITHUB_ENV"
-          echo "TAG=$(git describe --always --tags | grep -E '[0-9]*\.[0-9]*\.[0-9]*' || echo snapshot)" >> "$GITHUB_ENV"
-      - name: ensure clean assets dir
```
Comment (uncleDecart) [inline .github/workflows/assets.yml:51]: Why keep assets folder?

Comment (yash-zededa) [reply .github/workflows/assets.yml:51]: I have just reverted to the version of the file from the version https://github.com/lf-edge/eve/blob/0ee051623dcc26176c82ad012a8f29ca35fbbd9f/.github/workflows/assets.yml

Comment (uncleDecart) [reply .github/workflows/assets.yml:51]: nvm, we're reverting

**Review by uncleDecart — APPROVED** [review]
LGTM

**Pull request merged**